x-pack/test/detection_engine_api_integration/utils/install_prebuilt_rules_from_mocks.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+import { InstallPrebuiltRulesAndTimelinesResponse } from '@kbn/security-solution-plugin/common/detection_engine/prebuilt_rules';
+import type SuperTest from 'supertest';
+import {
+  createPrebuiltRuleAssetSavedObjects,
+  RulesPackageType,
+} from './create_prebuilt_rule_saved_objects';
+import { installPrebuiltRules } from './install_prebuilt_rules';
+
+/**
+ * Creates prebuilt rule mocks and installs them
+ *
+ * @param supertest Supertest instance
+ * @param es Elasticsearch client
+ * @returns Install prebuilt rules response
+ */
+export const installPrebuiltRulesFromMocks = async (
```

Comment:
> Are we going to have (or already have) a function like `installRealPrebuiltRules` or `installPrebuiltRulesFromRealAssets`?

Installing real prebuilt rules is needed only for a specific test case, so I doubt we need a separate helper for that. Moreover, it could be dangerous to give the ability to install the real rules package as that might be overused.